samples/highcharts/demo/area-inverted/demo.js

```diff
@@ -25,33 +26,28 @@ Highcharts.chart('container', {
             Highcharts.defaultOptions.legend.backgroundColor || '#FFFFFF'
     },
     xAxis: {
-        categories: [
-            'Monday',
-            'Tuesday',
-            'Wednesday',
-            'Thursday',
-            'Friday',
-            'Saturday',
-            'Sunday'
-        ]
+        categories: ['2014', '2015', '2016', '2017', '2018', '2019', '2020', '2021']
     },
     yAxis: {
-        title: {
-            text: 'Number of units'
+        labels: {
+            formatter(axis) {
+                return axis.value / 100 + ' B';
```

Comment:
Better give the data itself in Billons so we don't need the formatter (we prefer `format`). Also, the tooltip should state the units of the number.